srslie/fitlit
# Css

**Changes:**
- Update CSS styling and event listener functionality, including hover, clickability and media queries.  
- Add toggle functionality to category cards and graphs
- Refactor some functionality

**Issues:**
- Time constraints prevented us from implementing all of the accessibility issues we wanted to but were able to include a responsive design.

- [x] DRY and SRP (DRYish?)

**How to Review Changes or Manually Test Code:**
- open index.html
- check in scripts.js for code changes
- view features in browser
